Entities/Special/Dummy/Dummy.as

```diff
@@ -31,20 +30,32 @@ void onGib(CSprite@ this)
 		int r = (((blob.getNetworkID() + 1) * 31) % 3);
 		CParticle@ Large1   = makeGibParticle(filename, pos, vel + getRandomVelocity(90, hp - 0.2 , 80), 7 - (r % 2), 1 - (r / 2), Vec2f(16, 16), 2.0f, 20, "/material_drop", team);
 	}
+}
 
+bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
+{	
+    if (blob.getName() == "arrow" && this.getTeamNum() != blob.getTeamNum())
+    {
+        return true;
+    }
+    return false;
 }
 
 f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitterBlob, u8 customData)
 {
 	if (damage > 0.0f)
-	{
+	{	
+		CSprite@ sprite = this.getSprite();
+	
 		if (worldPoint.x > this.getPosition().x)
 		{
-			this.setAngleDegrees(this.getAngleDegrees() - 3 - XORRandom(10));
```

Comment:
Remove this line rather than commenting it out. Same with the other line lower down.

Reply:
Will do. I just wanted to convey this alternative way of doing it, too.
RotateBy() doesn't use angle, so getAngleDegrees() always returns 0. Therefore I cannot implement a threshold after which the Dummy no longer tilts further.